draft-ietf-quic-multipath.md

```diff
@@ -350,13 +350,15 @@ on different 4-tuples due to, e.g., NAT rebinding. In such a case, the receiver
 as specified in {{Section 9.3 of QUIC-TRANSPORT}} by initiating path validation
 but MUST use a new connection ID for the same Path ID.
 
-This proposal adds four multipath control frames for path management:
+This proposal adds five multipath control frames for path management:
 
 - PATH_ABANDON frame for the receiver side to abandon the path
 (see {{path-abandon-frame}}),
 - PATH_STANDBY and PATH_AVAILABLE frames to express a preference
 in path usage (see {{path-standby-frame}} and {{path-available-frame}}), and
-- MAX_PATH_ID frame for increasing the limit of active paths.
+- MAX_PATH_ID frame (see {{max-paths-frame}}) for increasing the limit of
+active paths, while MAX_PATH_ID_BLOCKED frame (see {{max-paths-blocked-frame}})
+indicates that the limit of active paths set by the peer has been reached.
```

Comment:
should it be stated that it is an request to the peer to consider increasing it?